examples/webgl_loader_bvh.html

```diff
@@ -47,7 +47,7 @@
 			const loader = new BVHLoader();
 			loader.load( 'models/bvh/pirouette.bvh', function ( result ) {
 
-				const skeletonHelper = new THREE.SkeletonHelper( result.skeleton.bones[ 0 ] );
+				const skeletonHelper = new THREE.SkeletonHelper( result.skeleton.bones[ 0 ], { axesHelperSize: 5 } );
```

Comment:
Since this feature is optional, please update only a single. I suggest use it in `webgl_loader_bvh.html`.